code/obj/item/implant.dm

```diff
@@ -469,7 +469,7 @@ THROWING DARTS
 					ticker.mode:remove_revolutionary(H.mind)
 				else
 					if (prob(30))
-						H.show_text("<B>The [src] burns and rattles inside your chest! It's attempting to force your loyalty!</B>", "blue")
+						H.show_text("<B>The [src] burns and rattles inside your chest! It's attempting to prevent you revolting!</B>", "blue")
```

Comment:
I like the original wording more here. From my POV the main issue with the current name is people assuming the implants will convert traitors, etc- if you're a rev, you already know they're bad for you, and we don't really need to worry about confusion

Reply:
Restored with a slight change to add loyalty to heads of staff.